App.js

```diff
@@ -2,15 +2,29 @@ import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 import { Tabs } from './src/lib/Router';
 import { AppLoading } from 'expo';
+import RootNavigation from './src/containers/RootNavigation';
 
 import { Provider } from 'react-redux';
 import Store from './src/lib/Store';
 
+import * as firebase from 'firebase';
+
+firebase.initializeApp({
+  apiKey: "AIzaSyBZGYJUNo7DuYgSUhCZ3S58zM0gO7fcYrY",
```

Comment:
This should never be pushed to GitHub. A commonly accepted practice for this is to grab this data from a config file (eg. `firebase.config`) which is included in `.gitignore`, but include a version of the config file `firebase.sample.config` which is stripped down so that it doesn't hold any private information. The `firebase.sample.config` would then be copied to `firebase.config` and filled out with the necessary information locally before compilation.